modules/network-observability-configuring-custom-metrics.adoc

```diff
@@ -0,0 +1,201 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-configuring-custom-metrics_{context}"]
+= Configuring custom metrics
+. In the web console, go to Operators → Installed Operators.
+. Under the Provided APIs heading for the NetObserv Operator, select Flow Collector.
+. Select cluster then select the YAML tab.
+. Edit the `FlowCollector`  to add the following parameters for `spec.processor`:
+  `processor.subnetLabels` field to create the FlowMetrics API.
++
+[source,yaml]
+—-
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+Spec:
+# …
+processor:
+  addZone: true                <1>
+  subnetLabels:
+    openShiftAutoDetect: true  <2>
+    customLabels:
+    - cidrs: []
+      name: ""
+# …
+—-
+<1> Optional: Configure the ability to see zones in the metrics data. 
+<2> spec.processor.subnetLabels.openShiftAutoDetect` set to `true` to enable custom metrics. 
+
+. Navigate to the Import icon, +, in the top right corner. 
+. Here are a few examples of `FlowMetrics` configurations you can use:
++
+.Identify traffic from unknown destination
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowMetric
+metadata:
+   name: flowmetric-pods-talking-outside
+   namespace: netobserv
+spec:
+   metricName: pods_talking_outside_bytes_total  <1>
+   type: Counter            <2>
+   valueField: Bytes
+   direction: Ingress       <3>
+   Labels: [SrcK8S_Name,SrcK8S_Namespace,SrcK8S_OwnerName,SrcK8S_OwnerType,DstAddr]       <4>
+   Filters:                <5>
+     - field: DstK8S_OwnerType
+       matchType: Absence
+     - field: SrcK8S_Type
+       value: Pod
+----
+<1> How the metrics will appear, with the prefix `netobserv`. 
+<2> Counter is useful for counting bytes or packets. Histogram is the other available type, which is useful for latencies. 
+<3> Direction of traffic to capture. Ingress is an alternative option. If not specified, both ingress and egress are captured. 
+<4> Labels define what the metrics look like and the relationship between the different entities. The labels also define the metrics cardinality. For example, `SrcK8S_Name` is a high cardinality metric.
+<5> Filters allow further refinement of what you want to see. In this example, the filters ultimately refine to show external traffic, as the name indicates pods talking outside of the cluster. The `field: DstK8S_OwnerType` and `matchType: Absence` means if there is an absence of recognized IP address from the OwnerType, it is not a known ID, so there is a good chance that it is external. 
+====
++
+.A-Z aware
```

Comment:
Honestly, I'd prefer to avoid linking documentation and actual examples too closely, as this could become a maintenance burden. In fact if you look at [this pull request](https://github.com/netobserv/network-observability-operator/pull/609) that I have, it already makes the doc outdated 😬
Perhaps we can take just 2 examples, one showing a Counter metric and another a Histogram metric, as they are made slightly differently.